server/services/healthCheck.ts

```diff
@@ -49,16 +49,8 @@ function gatherCheckInfo(aggregateStatus: Record<string, unknown>, currentStatus
   return { ...aggregateStatus, [currentStatus.name]: currentStatus.message }
 }
 
-const apiChecks = [
-  ...(config.apis.tokenVerification.enabled
-    ? [
-        service(
-          'tokenVerification',
-          `${config.apis.tokenVerification.url}/health/ping`,
-          config.apis.tokenVerification.agent,
-        ),
-      ]
-    : []),
+const apiChecks: HealthCheckService[] = [
```

Comment:
I know the health check in the template has changed a bit recently; will do a separate PR to pull in template changes at some point...